lnwire/lnwire.go

```diff
@@ -292,17 +292,47 @@ func writeElement(w io.Writer, element interface{}) error {
 		}
 
 	case *net.TCPAddr:
-		var ip [16]byte
-		copy(ip[:], e.IP.To16())
-		if _, err := w.Write(ip[:]); err != nil {
-			return err
-		}
+		if e.IP.To4() != nil {
+			var descriptor [1]byte
+			descriptor[0] = 1
```

Comment:
Can you lift the `1` and `2` values, into an `addrType` enum similar to what was done in the `channeldb` package? Thanks!